apps/native-component-list/src/screens/ExpoMaps/apple/MapsPolylineScreen.tsx

```diff
@@ -0,0 +1,371 @@
+import { AppleMaps } from 'expo-maps';
+import { Alert, View } from 'react-native';
+
+export default function MapsCameraControlsScreen() {
+  return (
+    <View style={{ flex: 1 }}>
+      <AppleMaps.View
+        style={{ width: 'auto', height: '100%' }}
+        cameraPosition={{
+          coordinates: {
+            latitude: 46.334295,
+            longitude: 16.5656,
+          },
+          zoom: 15,
+        }}
+        onPolylineClick={(event) => {
+          console.log(event);
+          Alert.alert('Polyline clicked', JSON.stringify(event));
+        }}
+        polylines={[
+          {
+            strokeColor: '#ff0000',
+            strokeWidth: 20,
+            coordinates: [
+              {
+                latitude: 46.334295,
+                longitude: 16.5656,
+              },
+              {
+                latitude: 46.334278,
+                longitude: 16.565874,
+              },
+              {
```

Comment:
Can we reuse that data between the iOS and Android implementations? Also, let's extract it to a separate file. It's making the code less readable right now.

Reply:
Sure, extracted and reused.